### DISCUSSION TITLE
unit test

### DISCUSSION THREAD

**Discussion by hmb1** [duckdb/duckdb#2928] [Q&A]
Am trying to run the   unit tests  **ossfuzz** and **sqlite**  but can't seem to get the command line right ? 
Am I missing something really obvious ?
e.g
**unittest test/ossfuzz 
unitttest ossfuzz**  
unittest -c ossfuzz

can run the  test/sql  directory OK 
e.g 
**unittest test/sql/catalog/function/test_subquery_macro.test**

Have tried above with both a Ninja build and Makefile build with the same results.

**Comment by Mytherin** [comment]
You can run a specific directory by placing the directory name between square brackets, e.g. `unittest "[ossfuzz]"`

**Comment by hmb1** [comment]
Many thanks Mark
That worked with everything:   [common], [api], [sqlite]  [extension]  
except [ossfuzz]   

must be some kind of build issue.
